### PR TITLE
review motivation section

### PR DESCRIPTION
**Pull request by niccokunzmann**

### CODE REVIEW

File: vendor/README.md [hunk 1]
```diff
@@ -16,12 +16,17 @@ should be included. They serve three purposes:
 Contributors should take care to cover each part of a badge's functionality,
 and ideally, all code branches:
 
-- Typical case
-- Customized cases
-- Server errors
-- Not found errors
+- Typical case like
+  - file is present/absent
```
Comment (paulmelnikow) [inline vendor/README.md]: I'm not sure "file absent" is a typical case. Could you clarify?

Would it be worth specifically mentioning something like "repo not found" (or "project not found")? That _should_ be a 404 according to the HTTP spec, but these services don't adhere to that.

File: vendor/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Vendor tests
 When creating a badge for a new service or changing a badge's behavior, tests
 should be included. They serve three purposes:
 
-1. The contributor and revewer can easily verify the code works as
+1. The contributor and reviewer can easily verify the code works as
```
Comment (paulmelnikow) [inline vendor/README.md]: Thanks!

File: vendor/README.md (hunk 1 above)
Comment (niccokunzmann) [inline vendor/README.md]: The LICENSE file is an example. But it might be of almost no importance.

Comment (paulmelnikow) [inline vendor/README.md]: Is there a badge you're thinking of which depends on the LICENSE file?

Comment (niccokunzmann) [inline vendor/README.md]: [license]()

    [license]()

Comment (niccokunzmann) [inline vendor/README.md]: But, maybe there are etter or more examples like build fails/passes

Comment (niccokunzmann) [inline vendor/README.md]: It could also be 

- tested/not tested/coverage
- 0, 1, 1000, 1000000, 1000000000 downloads

Comment (paulmelnikow) [inline vendor/README.md]: After I merged this, I reworked this list again and annotated the vendor code in 1002045. See what you think. Feel free to adjust further.

**Pull request merged**